src/antsibull_core/galaxy.py

```diff
@@ -7,9 +7,11 @@
 
 from __future__ import annotations
 
+import asyncio
 import os.path
 import shutil
 import typing as t
```

Comment:
The main reason is that this needs to be backported to stable-1, so it's easier to keep them and remove them later (after backporting). Otherwise I have to do the work twice.